kit/src/test/java/com/oracle/javafx/scenebuilder/kit/skeleton/SkeletonFileNameProposalTest.java

```diff
@@ -107,11 +103,11 @@ public void that_fxmlLocation_in_resources_dir_is_changed_to_java_specific_direc
     }
 
     @Test
-    public void that_fxmlLocation_in_resources_dir_is_changed_to_kotlin_specific_directory() throws Exception {
+    public void that_fxmlLocation_in_resources_dir_is_changed_to_kotlin_specific_directory(@TempDir Path temporaryDirectory) throws Exception {
```

Comment:
JUnit5 `@TempDir` is awesome, you can use it at any place where it is needed. It is no longer necessary to make it a field of the test class. It works fine as a method argument.